js/liveaw.js

```diff
@@ -76,11 +76,12 @@ function list_matches(s, cb) {
 }
 
 function on_edit_event(s) {
-	var match_players = {};
-	s.event.matches.forEach(function(match) {
+	var match_players = s.event.matches.reduce(function(prev, match) {
```

Comment:
The previous `forEach` iteration does nothing more than to reduce all values of the incoming array to a single object. That's why I think the `reduce` method is more fitting here.

If the initial object remains empty and is not mutated by the reduce, than that is a strong indicator that `s.event.matches` is in fact an empty array.